### ISSUE TITLE
Support for docker hosting

### ISSUE THREAD

**Issue by maybesmurf** [logchimp/logchimp#641]
Title says it all.

**Comment by laneparton** [comment]
As in, a dockerfile built for it? Wouldn't be too hard, would it?

**Comment by mittalyashu** [comment]
Yes, the Dockerfile will get quite large by including Postgresql database.

Dockerfile will only include the LogChimp source code and it can be run using docker-compose.

**Comment by frode** [comment]
If the app is not already setup with the following, this would solve any "large db" issues:

- Use a tool or code to run db migrations
- Thereafter seed the initial required data to run the app

What do you think @mittalyashu

**Comment by mittalyashu** [comment]
Happy to share that now you can deploy LogChimp using Docker images.

You can quickly sign up LogChimp using [docker compose](https://github.com/logchimp/logchimp/blob/master/docker/community/docker-compose.yml).

Closed by https://github.com/logchimp/logchimp/pull/959
